src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int sumOfBuying = 0;
+        int sumOfSupply = 0;
+        File inputFile = new File(fromFileName);
+        File outputFile = new File(toFileName);
+        try (BufferedReader reader = new BufferedReader(new FileReader(inputFile))) {
+            String lineFromFile = reader.readLine();
+            while (lineFromFile != null) {
+                String operationType = lineFromFile.substring(0,lineFromFile.indexOf(','));
```

Comment:
[Use split() instead of substring()](https://github.com/mate-academy/jv-work-with-file/blob/master/checklist.md#use-split-instead-of-substring)